jsx/Form.js

```diff
@@ -328,7 +328,7 @@ class SelectElement extends Component {
       }
     }
 
-    this.props.onUserInput(this.props.name, value);
+    this.props.onUserInput(this.props.name, e.target.value, e.target.id, 'select');
```

Comment:
does using `e.target.value` negate the purpose of lines 322-329?

Reply:
why does this use the `id` of the event target but `TextboxElement` uses the one passed by `props`?

Reply:
These are good questions.

Reply:
makes sense